src/main/java/XlsxExcelFile.java

```diff
@@ -81,8 +82,28 @@ public void writeCell(String content, int rowIdx, int columnIdx) {
             Logger.printError("Cell at column " + columnIdx + " does yet not exist. Creating new cell...");
             cell = row.createCell(columnIdx);
         }
-
-        cell.setCellValue(content);
+        
+		cell.setCellType(content.type);
+        switch(content.type) {
```

Comment:
How do you propose to handle the various cell types? I'm not sure java's polymorphism will correctly select the method based on an object return type that's an instance of Integer, Boolean or String. Unless there is a different way to return various different types with the same function.